.changeset/nice-shirts-fail.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/console": patch
+"@wso2is/form": patch
```

Comment:
Let's bump the dependent packages  of the form package as well

Reply:
You can find the dependents of a package by running `npx nx graph` in the root folder.

Reply:
This is verified. Only console needs the form compoent.